drake/examples/Cars/simulateLCM.cpp

```diff
@@ -0,0 +1,135 @@
+
+#include "LCMSystem.h"
+#include "RigidBodySystem.h"
+#include "LinearSystem.h"
+#include "BotVisualizer.h"
+#include "drakeAppUtil.h"
+#include "lcmtypes/drake/lcmt_driving_control_cmd_t.hpp"
+
+using namespace std;
+using namespace Eigen;
+using namespace Drake;
+
+template <typename ScalarType = double>
+class DrivingCommand {
+public:
+  typedef drake::lcmt_driving_control_cmd_t LCMMessageType;
+  static std::string channel() { return "DRIVING_COMMAND"; };
+
+  DrivingCommand(void) : throttle(0), brake(0), steering_angle(0) {};
+  template <typename Derived>
+  DrivingCommand(const Eigen::MatrixBase<Derived>& x) : steering_angle(x(0)), throttle(x(1)), brake(x(2)) {};
+
+  template <typename Derived>
+  DrivingCommand& operator=(const Eigen::MatrixBase<Derived>& x) {
+    steering_angle = x(0);
+    throttle = x(1);
+    brake = x(2);
+    return *this;
+  }
+
+  friend Eigen::Vector3d toEigen(const DrivingCommand<ScalarType>& vec) {
+    Eigen::Vector3d x;
+    x << vec.steering_angle, vec.throttle, vec.brake;
+    return x;
+  }
+
+  friend std::string getCoordinateName(const DrivingCommand<ScalarType>& vec, unsigned int index) {
+    switch (index) {
+      case 0: return "steering_angle";
+      case 1: return "throttle";
+      case 2: return "brake";
+    }
+    return "error";
+  }
+  const static int RowsAtCompileTime = 3;
+
+  ScalarType steering_angle;
+  ScalarType throttle;
+  ScalarType brake;
+};
+
+bool decode(const drake::lcmt_driving_control_cmd_t& msg, double& t, DrivingCommand<double>& x) {
```

Comment:
same as above.  other decode methods might fail, this one is so simple that failure is not considered an option.